README.adoc

```diff
@@ -11,6 +11,79 @@ This repository follows the monorepo pattern and includes the following projects
 - **verifier**: A command line application to audit ballots and verify the election results.
 - **voting_schemes**: Implementation of different cryptographic voting schemes to be used to run the elections. Currently, it includes the Dummy scheme, used only for testing purposes, and the ElectionGuard scheme, based on the Python implementation of the protocol.
 
-## License
+== Getting started with this repository
+
+- Clone this repository with submodules `git clone --recurse-submodules -j8 https://github.com/decidim/decidim-bulletin-board.git`
+- Or clone the submodules separately after the initial clone by running `git submodule update --init --recursive` in the repository folder
+- Use one of the development methods below
+
+=== Devcontainer
+
+This repository ships with a dev container to get you up and running quickly. This make is easy to get the development environment running without having to install all the dependencies.
+
+Read more about https://code.visualstudio.com/docs/remote/containers[devcontainers on Visual Studio Code documentation].
+
+=== Manual development environment
+
+==== Install Ruby and Node
+
+Assuming you already know how to if you are already working with Decidim.
+
+==== Install Python
+
+This guide is assuming that you don't have any python development tooling installed on your home folder. If so, we recommend that you start with a clean state just to be sure, for instance, deleting .local/bin/pip with rm $[HOME}/.local/bin/pip
+
+You can use e.g. https://github.com/pyenv/pyenv[pyenv] to install the correct version.
+
+- Run `curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash`
+- Add the following to your `~/.bashrc`:
+```
+export PYENV_ROOT="$HOME/.pyenv"
+export PATH="$PYENV_ROOT/bin:$PATH"
+eval "$(pyenv init --path)"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
+- Restart the console
+- Check the correct Python version by running `grep python_version voting_schemes/electionguard/python-wrapper/Pipfile`
+- Run `PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.x.x` (replace 3.x.x with the correct python version, note the configuration options)
+
+==== Install Poetry
+
+- Run `curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -`
```

Comment:
This installation method is deprecated. When running it, I have the following error: 

```bash
$ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python - 
Retrieving Poetry metadata

This installer is deprecated, and scheduled for removal from the Poetry repository on or after January 1, 2023.
See https://github.com/python-poetry/poetry/issues/6377 for details.

You should migrate to https://install.python-poetry.org instead, which supports all versions of Poetry, and allows for `self update` of versions 1.1.7 or newer.
Instructions are available at https://python-poetry.org/docs/#installation.

Without an explicit version, this installer will attempt to install the latest version of Poetry.
This installer cannot install Poetry 1.2.0a1 or newer (and installs will be unable to `self update` to 1.2.0a1 or newer).

To continue to use this deprecated installer, you must specify an explicit version with --version <version> or POETRY_VERSION=<version>.
Alternatively, if you wish to force this deprecated installer to use the latest installable release, set GET_POETRY_IGNORE_DEPRECATION=1.

Version 1.2.1 is not supported by this installer! Please specify a version prior to 1.2.0a1 to continue!

```

Now the official [installer](https://python-poetry.org/docs/#installing-with-the-official-installer) is: 
```suggestion
- Run `curl -sSL https://install.python-poetry.org | python3 -`
```